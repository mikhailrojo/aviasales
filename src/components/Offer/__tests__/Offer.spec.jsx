import * as React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';

import {Offer} from '../index';

const TEST_OFFER = {
	arrival_date: '12.05.18',
	arrival_time: '22:10',
	carrier: 'TK',
	departure_date: '12.05.18',
	departure_time: '16:20',
	destination: 'TLV',
	destination_name: 'Тель-Авив',
	origin: 'VVO',
	origin_name: 'Владивосток',
	price: 12400,
	stops: 3
};

describe('Snapshot', () => {
	it('render Offer with offer details', () => {
		const renderer = new ShallowRenderer();
		const component = renderer.render(
			<Offer
				{...TEST_OFFER}
				currency='RUB'
			/>
		);
		expect(component).toMatchSnapshot();
	});
});
