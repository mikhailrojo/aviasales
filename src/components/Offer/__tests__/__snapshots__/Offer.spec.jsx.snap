// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot render Offer with offer details 1`] = `
<styled.div>
  <LeftOfferPart
    carrier="TK"
    currency="RUB"
    price={12400}
  />
  <RightOfferPart
    arrival_date="12.05.18"
    arrival_time="22:10"
    departure_date="12.05.18"
    departure_time="16:20"
    destination="TLV"
    destination_name="Тель-Авив"
    origin="VVO"
    origin_name="Владивосток"
    stops={3}
  />
</styled.div>
`;
